python/tests/conftest.py

```diff
@@ -2,9 +2,10 @@
 import random
 import typing
 
+import py
 import pymongo
 import pytest
-from energuide import database
+from energuide import database, extractor
```

Comment:
Please put this on a separate line. Helps reduce merge conflicts and keeps diffs more focused.